apps/native/app/src/ui/lib/card/notification-card.tsx

```diff
@@ -81,13 +74,19 @@ export function NotificationCard({
   date,
   icon,
   unread,
+  underlayColor,
+  testID,
 }: CardProps) {
   const theme = useTheme()
+
   return (
     <Host
       onPress={() => onPress(id)}
-      underlayColor={theme.isDark ? theme.shade.shade200 : '#EBEBFA'}
+      underlayColor={
+        underlayColor ?? theme.isDark ? theme.shade.shade200 : '#EBEBFA'
```

Comment:
What is this light color that does not have a color variable and why do we need it here?